src/calculateRentalCost.js

```diff
@@ -28,7 +28,28 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const DAY_RENT_COST = 40;
+  const DISCOUNT_SHORT_TERM = 20;
+  const SHORT_TERM = 3;
+  const DISCOUNT_LONG_TERM = 50;
+  const LONG_TERM = 7;
+
+  let totalCost = 0;
+  const basePrice = DAY_RENT_COST * days;
+
+  if (days < SHORT_TERM) {
+    totalCost = basePrice;
+  }
+
+  if (days >= SHORT_TERM) {
+    totalCost = basePrice - DISCOUNT_SHORT_TERM;
+  }
+
+  if (days >= LONG_TERM) {
+    totalCost = basePrice - DISCOUNT_LONG_TERM;
+  }
```

Comment:
<img width="219" alt="image" src="https://github.com/mate-academy/js_task-transportation-on-vacation/assets/70220667/7afc4d2c-a712-439c-8a6e-a1f1fba1a25c">


```suggestion
  if (days >= SHORT_TERM) {
    return basePrice - DISCOUNT_SHORT_TERM;
  }

  if (days >= LONG_TERM) {
    return basePrice - DISCOUNT_LONG_TERM;
  }
```